client/src/context/useAuthContext.tsx

```diff
@@ -58,7 +58,7 @@ export const AuthProvider: FunctionComponent = ({ children }): JSX.Element => {
           // don't need to provide error feedback as this just means user doesn't have saved cookies or the cookies have not been authenticated on the backend
           setLoggedInUser(null);
           if (!(history.location.pathname === '/signup')) {
-            history.push('/login');
+            history.push('/');
```

Comment:
How come we are redirecting back to the landing page rather than the login page if the user isn't signed in? Let's revert this change 😃